runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -478,12 +480,13 @@ public AsyncBufferedImage getImage(int itemId, int quantity, boolean stackable)
 	 * @param itemId item id
 	 * @param itemQuantity item quantity
 	 * @param outlineColor outline color
+	 * @param thickness thickness of outline
 	 * @return image
 	 */
-	private BufferedImage loadItemOutline(final int itemId, final int itemQuantity, final Color outlineColor)
+	private BufferedImage loadItemOutline(final int itemId, final int itemQuantity, final Color outlineColor, final int thickness)
 	{
-		final SpritePixels itemSprite = client.createItemSprite(itemId, itemQuantity, 1, 0, 0, true, 710);
-		return itemSprite.toBufferedOutline(outlineColor);
+		final BufferedImage outline = ImageUtil.outlineImage(getImage(itemId, itemQuantity, false), outlineColor, false, thickness);
+		return outline;
```

Comment:
```suggestion
		return ImageUtil.outlineImage(getImage(itemId, itemQuantity), outlineColor, false, thickness);
```

The default for `outlineCorners` is false already. This also does not need to be saved to a variable since it is simply returned on the next line.

Reply:
the `false` you removed is for `stackable`, not `outlineCorners`
stackable should be false so the quantity text doesn't get outlined as well